Tests/Datagrid/ProxyQueryTest.php

```diff
@@ -45,46 +46,36 @@ public function dataGetFixedQueryBuilder()
      */
     public function testGetFixedQueryBuilder($class, $alias, $id, $expectedId, $value, $identifierType)
     {
-        $meta = $this->getMockBuilder('Doctrine\ORM\Mapping\ClassMetadataInfo')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $meta = $this->createMock('Doctrine\ORM\Mapping\ClassMetadataInfo');
         $meta->expects($this->any())
             ->method('getIdentifierFieldNames')
             ->willReturn(array($id));
         $meta->expects($this->any())
             ->method('getTypeOfField')
             ->willReturn($identifierType);
 
-        $mf = $this->getMockBuilder('Doctrine\ORM\Mapping\ClassMetadataFactory')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $mf = $this->createMock('Doctrine\ORM\Mapping\ClassMetadataFactory');
         $mf->expects($this->any())
             ->method('getMetadataFor')
             ->with($this->equalTo($class))
             ->willReturn($meta);
 
-        $platform = $this->getMockBuilder('Doctrine\DBAL\Platforms\PostgreSqlPlatform')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $platform = $this->createMock('Doctrine\DBAL\Platforms\PostgreSqlPlatform');
 
-        $conn = $this->getMockBuilder('Doctrine\DBAL\Connection')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $conn = $this->createMock('Doctrine\DBAL\Connection');
         $conn->expects($this->any())
             ->method('getDatabasePlatform')
             ->willReturn($platform);
 
-        $em = $this->getMockBuilder('Doctrine\ORM\EntityManager')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $em = $this->createMock('Doctrine\ORM\EntityManager');
         $em->expects($this->any())
             ->method('getMetadataFactory')
             ->willReturn($mf);
         $em->expects($this->any())
             ->method('getConnection')
             ->willReturn($conn);
 
-        $q = $this->getMock('PDOStatement');
+        $q = $this->createMock('PDOStatement');
```

Comment:
@OskarStark @jordisala1991 can either of you fix this or just not change the test that fails? That way we can benefit from it in other PRs